ni/measurementlink/discovery/v1/discovery_service.proto

```diff
@@ -176,4 +180,29 @@ message EnumerateComputeNodesRequest {
 message EnumerateComputeNodesResponse {
   // The list of compute nodes that have registered themselves in the current session.
   repeated ComputeNodeDescriptor compute_nodes = 1;
+}
+
+message ResolveServiceWithInformationRequest {
```

Comment:
We can reuse the `ResolveServiceRequest`

Reply:
Removed `ResolveServiceWithInformationRequest` and resued `ResolveServiceRequest`.